docs/dev/building.md

```diff
@@ -22,37 +22,44 @@ git clone https://github.com/scylladb/scylla
 cd ./scylla
 git submodule update --init --force --recursive
 ./tools/toolchain/dbuild ./configure.py --mode=<mode>
-./tools/toolchain/dbuild ninja
+./tools/toolchain/dbuild cmake --build build
 ```
 
 where `mode` is `dev` for development builds, `release` for release builds, and `debug` for debug builds.
 
-Alternatively, you can also pass the `mode` directly to the `ninja` command:
+Alternatively, you can also pass the `config` directly to the `cmake` command:
 
 ```console
 git submodule update --init --force --recursive
 ./tools/toolchain/dbuild ./configure.py
-./tools/toolchain/dbuild ninja <mode>
+./tools/toolchain/dbuild cmake --build build --config <config>
 ```
 
-After the build completes, you can find build artifacts in `build/<mode>` directory.
+Instead of using `cmake`, you can use `ninja` directly with the notation of
+`<target>:<config>`:
+
+```console
+./tools/toolchain/dbuild ninja -C build tests:<config>
+```
+
+After the build completes, you can find build artifacts in `build/<config>` directory.
 
 You can run unit tests with:
 
 ```console
-./tools/toolchain/dbuild ninja test
+./tools/toolchain/dbuild ninja -C build test
```

Comment:
I think this is not enough - I almost never want to build, let alone, all tests, and want to compile a single test. So I need to know a command line like:
```
ninja -C build test/boost/Dev/sstable_datafile_test
```
I would have also liked to see an explanation of where cmake puts object files (like `test/boost/CMakeFiles/sstable_datafile_test.dir/Debug/sstable_datafile_test.cc.o` - who can guess that???) but I guess it doesn't belong in this specific place. But I think it needs to appear somewhere. Developers will need to understand the new organization of build/, which is very different from what it was.

Reply:
i think "Getting Started" should provide the minimal information to get a developer started. i will update the "Testing" section with more details on how to build a single test.